libs/wingc/src/type_check/jsii_importer.rs

```diff
@@ -104,9 +104,38 @@ impl<'a> JsiiImporter<'a> {
 				} else {
 					self.lookup_or_create_type(type_fqn)
 				}
-			} else if let Some(Value::Object(_)) = obj.get("collection") {
-				// TODO: handle JSII to Wing collection type conversion, for now return any
-				self.wing_types.anything()
+			} else if let Some(Value::Object(d)) = obj.get("collection") {
+				let collection_kind = d
+					.get("kind")
+					.expect("'kind' is required for collection types")
+					.as_str()
+					.expect("'kind' must be a string");
+
+				let element_type = d
+					.get("elementtype")
+					.expect("'elementtype' is required for collection types")
+					.as_object()
+					.expect("'elementtype' must be an object");
+
+				// TODO: Handle non-primitive collections
+				let primitive_type = element_type
+					.get("primitive")
+					.expect("'primitive' is required for collection element types")
```

Comment:
Should the error message be something like "non-primitive collection types not yet supported"?

Reply:
done, good callout